QGIS_SERVER_DISABLE_GETPRINT setting environment (#6044)
qgis/QGIS-Documentation: docs/server_manual/config.rst

Do not use it if layers' extent is not fixed during the project's use.

QGIS_SERVER_DISABLE_GETPRINT
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> This is an option at the project level to improve project read time by disabling the
> layouts load.
> 
> This option disables WMS GetPrint request and activates thes QGIS project's flag to do not load layouts.

Reviewer: ```suggestion
This is an option at the project level to improve project read time by disabling
loading of layouts.

This option disables the WMS GetPrint request.
Activate this QGIS project flag to not load layouts.
```
